tuning/config/acceleration_configs/__init__.py

```diff
@@ -18,3 +18,11 @@
 from .fast_moe import FastMoeConfig
 from .fused_ops_and_kernels import FusedOpsAndKernelsConfig
 from .quantized_lora_config import QuantizedLoraConfig
+
+
+def get_more_accel_framework_callbacks(active_plugins, **kwargs):
```

Comment:
nit: could you place this in `utils` or a different file?

also possibly rename to `get_additional_accel_framework_callbacks`